app/models/manageiq/providers/kubernetes/container_manager.rb

```diff
@@ -41,6 +41,10 @@ def self.description
     @description ||= "Kubernetes".freeze
   end
 
+  def self.fonticon
+    @fonticon ||= "pficon-kubernetes".freeze
```

Comment:
I'm not sure that's better.. assuming we can force our names on patternfly, we'd still need a way to determine if that icon actually exists or not. (As in, there is currently no `pficon-ec2`, and the icons for openshift and openshift_enterprise should probably be the same, ...)